wiki/Tournaments/OMIC/2022_4K/id.md

```diff
@@ -0,0 +1,420 @@
+
+- FreeMod
+  1. [Slappy - Hanami (RuleBlazing) \[Edit\]](https://osu.ppy.sh/beatmapsets/1101411#mania/2329945)
+  2. [The Tech Thieves - Fake (Erkfir) \[Faked It (Edit)\]](https://osu.ppy.sh/beatmapsets/1048516#mania/2259015)
+  3. [cillia - FIRST (juankristal) \[MX\]](https://osu.ppy.sh/beatmapsets/512408#mania/1088936)
+  4. [Mafumafu - Tachiiri Kinshi (Raveille) \[No Entry\]](https://osu.ppy.sh/beatmapsets/987136#mania/2064918)
+  5. [rejection - Aimai Attitude (feat. Nakamura Sanso) (komugii) \[Obscure Love\]](https://osu.ppy.sh/beatmapsets/1381611#mania/3498128)
+  6. [Kommisar - Yamada Stream (DannyPX) \[edit 1.1x (280bpm)\]](https://osu.ppy.sh/beatmapsets/1167743#mania/2842360)
+  7. [Night Tempo - Nemure (Valedict) \[Rest\]](https://osu.ppy.sh/beatmapsets/562937#mania/1190462)
+  8. [The Ghost of 3.13 - Path To Nowhere (Shoegazer) \[Hopeless\]](https://osu.ppy.sh/beatmapsets/422800#mania/913882)
+  9. [PSYQUI - Eyes on me feat. Such (HowToPlayLN) \[Stage 4: Focused\]](https://osu.ppy.sh/beatmapsets/1196939#mania/2493320)
+  10. [ARM feat.Nicole Curry - Come to Life (HowToPlayLN) \[Inspiration\]](https://osu.ppy.sh/beatmapsets/1117755#mania/2334914)
+- Tiebreaker
+  1. **[Camellia - liquated (Eclipse-) \[gragh grag\]](https://osu.ppy.sh/beatmapsets/1174195#mania/2448995)**
+
+### Kualifikasi
+
+**[Unduh mappack di sini! (21 MB)](https://drive.google.com/file/d/1FbXVE1THzf6MDHeU0uSRJC2X6m9pt-T_/view?usp=sharing)**
+
+- FreeMod
+  1. [LeaF - MARENOL 1mg (Angki) \[Stage 1: Hallucination\]](https://osu.ppy.sh/beatmapsets/1793984#mania/3676731)
+  2. [Raytoly - Back To You ~Raytoly's Together Remix~ (Antipole) \[Stage 2: Solitude\]](https://osu.ppy.sh/beatmapsets/1793980#mania/3676726)
+  3. [succducc - me & u (Kara Edit) (TingMomentum) \[Stage 3: Affection\]](https://osu.ppy.sh/beatmapsets/1794198#mania/3677199)
+  4. [Laur - Exitium (zutu) \[Stage 4: Destruction\]](https://osu.ppy.sh/beatmapsets/1793987#mania/3676738)
+  5. [USAO & Camellia - Mobius (FelixSpade) \[Stage 5: Ouroboros\]](https://osu.ppy.sh/beatmapsets/1793983#mania/3676730)
+
+## Hasil Pertandingan
+
+<!---
+### Grand Final
+
+Sabtu, 20 Agustus 2022
+
+| Bracket | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Loser Final | Onta\_Bekasi ::{ flag=ID }:: | 0 | 0 | ::{ flag=ID }:: \[Albert\] | []() |
+
+Minggu, 21 Agustus 2022
+
+| Bracket | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Grand Final | RetroEX ::{ flag=ID }:: | 0 | 0 | ::{ flag=ID }:: Winner of 61 | []() |
+| Bracket Reset | Winner of 62 ::{ flag=ID }:: | 0 | 0 | ::{ flag=ID }:: Loser of 62 | []() |
+
+--->
+### Final
+
+Sabtu, 6 Agustus 2022
+
+| Bracket | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Loser | adihza ::{ flag=ID }:: | -1 | **0** | ::{ flag=ID }:: **Proxae** | [WBD](https://cdn.discordapp.com/attachments/977847617180540959/1005504567523430560/Screenshot_2022-08-06-23-56-01-97.jpg) |
```

Comment:
```suggestion
| Loser | adihza ::{ flag=ID }:: | -1 | **0** | ::{ flag=ID }:: **Proxae** | *win by default* |
```

Reply:
(don't think the screenshot is needed tbh; a simple `*win by default*` disclaimer should be more than enough)